lib/containers/engine.pm

```diff
@@ -201,17 +201,16 @@ sub info {
     $self->_engine_assert_script_run(sprintf("info %s %s", $property, $expected));
 }
 
-=head2 get_container_logs($container)
+=head2 get_container_logs($container, $filename)
 
 Request container's logs.
 C<container> the running container.
-C<logs> returns a string.
+C<filename> file the logs are written to.
 
 =cut
 sub get_container_logs {
-    my ($self, $container) = @_;
-    my $logs = $self->_engine_script_output("container logs $container");
-    return $logs;
+    my ($self, $container, $filename) = @_;
+    $self->_engine_assert_script_run("container logs $container | tee $filename");
```

Comment:
i would record_info in this case to make clear what have been done underneath the `get_container_logs` because the name doesnt implies that the logs will be found in `$filename`. or change the name if you dont mind?? wdyt?

Reply:
What do you suggest as new name?

Reply:
Alternatively, the logs could be handled entirely by the test code and not go through `grep` etc. on the SUT.

Reply:
But we were not doing the record_info before right? Not sure why it's needed now.

I don't mind about `$filename`... but if you want it more accurate, `$log_file` ?b

Reply:
> What do you suggest as new name?

i dont want to force this (my comment was a humble observation). but i am gonna give a suggestion for anycase. i would say `write_log` or something in this "line"